PocketApp.swift

```diff
@@ -7,11 +7,20 @@ import SwiftUI
 
 @main
 struct PocketApp: App {
+    @Environment(\.scenePhase)
+    var scenePhase
+
     @UIApplicationDelegateAdaptor var delegate: PocketAppDelegate
 
     var body: some Scene {
         WindowGroup {
             RootView(model: RootViewModel())
+        }.onChange(of: scenePhase) { newValue in
+            switch newValue {
```

Comment:
Unless I'm missing something, I don't think we have access to that, since that's a part of `UISceneDelegate`. But, since we're using the SwiftUI view lifecycle, we don't have a scene delegate.

Reply:
Misunderstood! Addressed.